modules/fame_azure-storage-file-backup/variables-gen.tf

```diff
@@ -0,0 +1,61 @@
+# backup detector
+
+variable "backup_notifications" {
+  description = "Notification recipients list per severity overridden for backup detector"
+  type        = map(list(string))
+  default     = {}
+}
+
+variable "backup_aggregation_function" {
+  description = "Aggregation function and group by for backup detector (i.e. \".mean(by=['host'])\")"
+  type        = string
+  default     = ""
```

Comment:
I don't remember well, empty means aggregate by all available dimensions?

Reply:
That's right